#This message allows to store data to be shared over different agents/base station
uint16 packet_size
hardwareInfo hardware_info
goalKeeperInfo agent_info
bool is_goalkeeper
